markdown-scheme_v3.Rmd

```diff
@@ -0,0 +1,393 @@
+        \item Working Paper: FORMAT
+          \begin{lstlisting}
+            RR_CountryCodeIso3_Year_PRWP-PRWPno_versionno
+            RR_CRI_2018_PRWP-8525_v01
+          \end{lstlisting}
+        \end{enumerate}
+  \item Other identifiers: DOI and https://doi.org/number
+  \item Title: Of the following form
+    \begin{lstlisting}
+      Reproducibility package for TITEL(paper)
+    \end{lstlisting}
+  \item Subtitle:
+    \begin{itemize}
+      \item WP: World Bank Policy Research Working Paper (PRWP) number
+      \item PP: DOI number (no https)
+    \end{itemize}
+\end{enumerate}
+
+## Authors and contributors
+Sufficient are
+\begin{enumerate}
+    \item Authoring entity: author, affiliation and ORCID number and email when available; For world bank authors use \textbf{World Bank}. Example:
+      \begin{verbatim}
+        Christoph Lakner World Bank ORCID 0000, clakner@worldbank.org
+      \end{verbatim}
+    \item Contributors: like RAs (not conference participants). Also coeditors if specifically mentioned. FORMAT
+    \begin{lstlisting}
+      Coeditor
+    \end{lstlisting}
+    \item Sponsors: All financial contributions and individual/general grants
+    \item Acknowledgements: Usually empty; Example: Corporate Taxation: special mentioning ministry employees for data assistance (could go to acknowledgment statement)
+    \item Acknowledgement Statement:
+      \begin{verbatim}
+        We are grateful to Benu Bidani, Francisco Ferreira, Luis-Felipe..
+      \end{verbatim}
+      \begin{verbatim}
+        .. and seminar participants at the World Bank’s Poverty ...
+      \end{verbatim}
+  \end{enumerate}
+
+## Version statement
+Sufficient are
+\begin{enumerate}
+    \item Project completion date: FORMAT: YEAR-MONTH
+      \begin{itemize}
+        \item PP: Date at which the article was published: Year-Month
+        \item WP: \textcolor{red}{Must be supplied, bc it is needed for the front page
+        of cataloge}
+      \end{itemize}
+    \item Version date: FORMAT: YEAR-MONTH-DAY
+      \begin{itemize}
+        \item PP: empty
+        \item WP: data of publication as research paper in WB Year-Month-Day
+      \end{itemize}
+\end{enumerate}
+
+## Scope and Coverage
+Sufficient are
+\begin{enumerate}
+    \item Abstract: Abstract of the article; \textcolor{red}{Hyphens are a problem. Check all abstracts. Also carriage returns are a problem, direct copy pasting leads to bad results}
+    \item Geographic areas: Format: Name, Code, Type
+    \begin{lstlisting}
+      Lebanon, LBN, ISO-3166 Alpha-3
+    \end{lstlisting}
+    \item Keywords: No keywords if none are given; Yes if given.
+    \item JEL codes:
+      \begin{verbatim}
+        ID D63
+        Name Equity, Justice, Inequality
+        Parent ID (empty)
+        Vocabulary JEL Classifications
+        URL https://www.aeaweb.org/econlit/jelCodes.php?view=jel
+      \end{verbatim}
+\end{enumerate}
+
+## Processes and Output
+The following fields exist
```

Comment:
want to focus on minimum required set of fields not what fields exist

Reply:
Yes, but here we have more possibilities, this is why I added the description for the items we are not currently using, which is Process, Institutional Review board, approval process and Reviews and comments. In theory, if I understand the description correctly, one could describe the lifecycle from start to end of the process. We do not use this yet. It is not obvious how we could use it.

Reply:
for now we want to focus on establishing the minimum necessary fields that all catalogue entries must have. it's fine if other fields are included, but those should be clearly marked as optional (and unless they are clearly useful i would be inclined not to include in our template ...)